packages/esm-appointments-app/src/routes.json

```diff
@@ -117,7 +117,7 @@
     },
     {
       "name": "edit-appointments-form",
-      "component": "appointementForm",
+      "component": "appointmentsForm",
```

Comment:
Is the convention to use plural? I know a form is used for a single appointment.

Reply:
Because the concern is the management of appointments, I think it makes sense to keep the nomenclature plural.